searchlib/src/vespa/searchlib/attribute/enumstore.h

```diff
@@ -74,7 +74,7 @@ class EnumStoreT : public IEnumStore {
     ssize_t load_unique_value(const void* src, size_t available, Index& idx);
 
 public:
-    EnumStoreT(bool has_postings, search::DictionaryConfig::Type type);
+    EnumStoreT(bool has_postings, const search::DictionaryConfig & type);
```

Comment:
Consider renaming parameter name to `dict_cfg`, to match definition in`enumstore.hpp`.

Reply:
Done, and then some